po/pl_PL.po

```diff
@@ -2029,7 +2032,7 @@ msgid "Other Expenses"
 msgstr "Inne wydatki"
 
 #: db/Table_Category.h:120 model/Model_Billsdeposits.cpp:33
-#: model/Model_Checking.cpp:32 transdialog.cpp:335
```

Comment:
hmmmm?

Reply:
But it should be included. There is `-kwxGetTranslation` option in update-po-files.sh script.